### PR TITLE
fix(concatjs): update karma to 6.3.2 and fix #2093

### LINKED ISSUE #2093: [karma-server]: UncaughtException:: file.detectType is not a function
Please see the Karma project's https://github.com/karma-runner/karma/issues/3537#, where they say the issue is with rules_nodejs.

### PR DESCRIPTION
**Pull request by mattsoulanille**
## PR Checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
No new tests added, but one was fixed to work with the change.
- [ ] Docs have been added / updated (for bug fixes / features)
Docs shouldn't need to be updated for this change.

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #2093 

## What is the new behavior?

Fixes #2093. Updates karma packages to latest versions.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

The old version of karma seems to still work with these changes (n = 2 tests), but I haven't extensively tested it.

### CODE REVIEW

File: packages/concatjs/index.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@
  */
 import * as crypto from 'crypto';
 import * as fs from 'fs';
+import * as File from 'karma/lib/file';
```
Comment (mattsoulanille) [inline packages/concatjs/index.ts:7]: This is somewhat hacky. We're probably not supposed to import from `lib`

**Comment by mattem** [comment]
Should we be bumping the peer dep here: https://github.com/bazelbuild/rules_nodejs/blob/stable/packages/concatjs/package.json#L25 ? Or this this API available in 4.0.0 too

**Comment by mattsoulanille** [comment]
Nice catch. I forgot that there are separate `package.json` files for the `pkg_npm` targets. 

We might not actually have to bump the peer dependency. The only key missing in the new [File](https://github.com/karma-runner/karma/blob/master/lib/file.js) object that is present in the current implementation's object is `content`, which the current one sets to the empty string. Karma 4 seems to work fine without it, at least with our tests in `//packages/concatjs/web_test/test/karma_typescript/...` and `//packages/concatjs/web_test/test/karma/...`.

We could still bump it to 5.1.0 just in case. What do you think?

**Comment by mattem** [comment]
As the tests pass I think this shows it's backwards compatible - we could bump the version as part of the upcoming 4.0.0 milestone release if we wanted, then it's a clear breaking point.

**Review by mattem — APPROVED** [review]

**Comment by mattsoulanille** [comment]
Sounds good to me. Thanks!

**Pull request merged**